src/core/Clock.js

```diff
@@ -1,3 +1,9 @@
+function now() {
+
+	return ( typeof performance === 'undefined' ? Date : performance ).now(); // see #10732
+
+}
+
```

Comment:
It looks a bit strange to put this function on top of the file. Please move it after the class definition.